content/docs/hooks-faq.md

```diff
@@ -362,6 +365,44 @@ It's possible that in the future React will provide a `usePrevious` Hook out of
 
 See also [the recommended pattern for derived state](#how-do-i-implement-getderivedstatefromprops).
 
+### Why am I seeing stale props or state inside my function? {#why-am-i-seeing-stale-props-or-state-inside-my-function}
+
+Any function inside a component, including event handlers and effects, “sees” the props and state from the render it was defined in. For example, consider code like this:
```

Comment:
A few suggestions.
* "sees" is clear enough for me, but due to the quotes maybe a little imprecise. 
* Functions are defined once, but are potentially created multiple times (during renders).

```suggestion
Any function inside a component, including event handlers and effects, contain ("sees") the props and state from the render it was created in. For example, consider code like this:
```